eng/CredScanSuppression.json

```diff
@@ -39,6 +39,10 @@
             "placeholder": "JdppJP5eH1w/CQ0cx4RGYWoC7NmQ0nmDbYR2PYWSDTXojV9bI1ck0Eh0sUIg8xj4KYj7tv+ZPLICu3BgLt6mMz==",
             "_justification": "Mocked key used in performance testing."
         },
+        {
+            "placeholder": "OAuth2ClientCredential",
+            "_justification": "This is not a credential key"
```

Comment:
Did it support limit the scope to that particular file?

Reply:
We can either define a fine to suppress, or define a string to suppress. https://github.com/Azure/azure-sdk-for-java/blob/main/doc/dev/CredScan-Process.md#false-positives

Reply:
I mean can we add a `file` (so the suppression only apply to the file)?